overrides/android-override.json

```
@@ -417,7 +417,20 @@
                         "formTypeSettings": [],
                         "inputTypeSettings": [],
                         "formBoundarySelector": "",
+                        "failsafeSettings": {},
                         "domains": [
+                            {
+                                "domain": ["coolors.co"],
+                                "patchSettings": [
+                                    {
+                                        "path": "/failsafeSettings",
+                                        "op": "replace",
+                                        "value": {
+                                            "maxInputsPerPage": 110
+                                        }
+                                    }
```

Add is much safer in terms of longevity here :)

```suggestion
                                "domain": ["coolors.co"],
                                "patchSettings": [
                                    {
                                        "path": "/failsafeSettings/maxInputsPerPage",
                                        "op": "add",
                                        "value": 10
                                    }
```